Database/Corrections/sodItemFixes.lua

```diff
@@ -32,6 +32,9 @@ function SeasonOfDiscovery:LoadItems()
         [209846] = {
             [itemKeys.objectDrops] = {409692},
         },
+        [210187] = {
+            [itemKeys.startQuest] = {78270},
```

Comment:
```suggestion
            [itemKeys.startQuest] = 78270,
```

Just noticed this needs to be a number, not a table.

You can verify that by using the "Recompile Database" function in the Advanced settings. That will scream at you when something is really broken.